public/docs/dart/latest/tutorial/toh-pt5.jade

```diff
@@ -48,648 +48,658 @@ p Run the #[+liveExampleLink2('', 'toh-5')] for this part.
   Start the Dart compiler, watch for changes, and start our server by entering the command:
 
 code-example(language="bash").
-    pub serve
+  pub serve
 
 :marked
   The application runs and updates automatically as we continue to build the Tour of Heroes.
 
-    ## Action plan
-    Here's our plan
+  ## Action plan
+  Here's our plan
 
-    * turn `AppComponent` into an application shell that only handles navigation.
-    * relocate the *Heroes* concerns within the current `AppComponent` to a separate `HeroesComponent`
-    * add routing
-    * create a new `DashboardComponent`
-    * tie the *Dashboard* into the navigation structure.
+  * turn `AppComponent` into an application shell that only handles navigation,
```

Comment:
If we're editing this, I'd use sentence caps but **not** add commas at the end. (Periods would be fine, though.) I'd also add a colon after "plan". For example:

Here's our plan:

* Turn ... navigation.
* ...
* Tie ... structure.